frontend-style.md

```diff
@@ -0,0 +1,351 @@
+# Frontend Style Guide
+
+## Structure
+
+## File organization
+
+All files should be organized by feature (e.g. math standard practice) instead of function (e.g. backbone collection). This includes:
+
+- Backbone models & collections
+- Components
+- Jest tests
+- Data mocks for jest tests
+
+Files pertaining to a single component specific to a feature should be organized as follows:
+
+```
+.
+└── my-page
+    ├── index.js
+    └── my-component
+        ├── index.js
+        ├── my-component.js
+        ├── my-component.scss
+        └── my-component.test.js
+```
+
+- `my-page/index.js` is a "handler" style component responsible for global
+  state management and API calls
```

Comment:
Usually, the "handler" is only responsible for gathering the data (usually from API calls). The page-level global state management bit is the responsibility of the "container".